PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -447,8 +452,14 @@
                 wireframe.inputType = "button"
                 wireframe.disabled = !button.isEnabled
 
+                // UIButton has a UILabel subview that displays the button's text
+                // We want to skip visiting children if button is sensitive
+                // Otherwise, we capture the button's title through its child UILabel
```

Comment:
but then we will check if the label is `isLabelSensitive` and it will mask anyway, right? confused...

Reply:
maybe a better approach would be to do what we do in the `findMaskableWidgets` and `maskChildren` flag, we still traverse the children, but respect the parent's masking.
the point is that the children may have more wireframes to be laid out and we will lose the details, eg if the `font` is applied to the UILable and not the Button